src/components/Main/Main.jsx

```diff
@@ -0,0 +1,15 @@
+import './main.css'
+
+export function Main() {
+    return (
+        <main className='main__total-block'>
+        <div className='main-block'>
+            <h1>Jump into learning for
+            less</h1>
+            <p className="main-info">If you're new to Udemy, we've got good news: For a
+limited time, courses start at just €14.99 for new
+learners! Shop now.</p>
+        </div>
+        </main>
+    )
```

Comment:
намагайся робити відступи скрізь правильні, бо потім тяжко буде

```suggestion
        <main className='main__total-block'>
           <div className='main-block'>
               <h1>Jump into learning for less</h1>
               <p className="main-info">
                   If you're new to Udemy, we've got good news: For a limited time, courses start at just €14.99 for new learners! Shop now.
                   </p>
           </div>
        </main>
    )
```